taccsite_cms/settings.py

```diff
@@ -234,15 +287,10 @@ def getsecrets():
     'djangocms_bootstrap4.contrib.bootstrap4_utilities',
     'haystack',
     'aldryn_apphooks_config',
-    # For faster testing, disable migrations during database creation
-    # SEE: https://stackoverflow.com/a/37150997
     'test_without_migrations',
     'taccsite_cms',
-    # Restore djangocms plugins that bootstrap4 hides
     'taccsite_cms.contrib.bootstrap4_djangocms_link',
     'taccsite_cms.contrib.bootstrap4_djangocms_picture',
-    # TODO: Extract TACC CMS UI components into pip-installable plugins
-    # FAQ: The djangocms_bootstrap4 library can serve as an example
     'taccsite_cms.contrib.taccsite_sample',
     'taccsite_cms.contrib.taccsite_system_monitor',
     'taccsite_cms.contrib.taccsite_data_list'
```

Comment:
❓ Unexpected isolated removal of individual comments.

1. What is the logic for deciding which comments to retain?

<details><summary>Notes</summary>

- Other comments about intent or details of the code below (like the `get_subdirs_as_module_names` function comments just below in this file) have **not** been removed.
- I have found that unused settings (hints) seem to be consistently removed.

</details>

Reply:
@xandroc I still believe some of these comments are useful. Where else do you suggest this information go?

```
    # For faster testing, disable migrations during database creation
    # SEE: https://stackoverflow.com/a/37150997
    'test_without_migrations',
```
```
    # Restore djangocms plugins that bootstrap4 hides
    'taccsite_cms.contrib.bootstrap4_djangocms_…',
```

Reply:
Revisiting in #343.